### PR TITLE
Order list

### PR DESCRIPTION
**Pull request by lcko1012**
## Related Tickets
- [#46933](https://edu-redmine.sun-asterisk.vn/issues/46933)

## WHAT (optional)
- Display order list of user

## HOW
- I add `index` function which load all orders of user to `orders_controller.rb` file
- Create `index.html.erb` file to display them

## Evidence (Screenshot or Video)
image

image

### CODE REVIEW

**Review by trinhdvt — APPROVED** [review]
ok

**Comment by lcko1012** [comment]
ready

File: app/views/orders/_order.html.erb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<div class="product-wrapper">
+  <div class="header">
+    <%= t "statuses.#{order.status}" %>
+  </div>
+
+  <div class="product">
+    <div>
+      <div class="details">
+        <img src=<%= first_prod(order).image %>
+        alt=<%= first_prod(order).name %>>
```
Comment (ghost) [inline app/views/orders/_order.html.erb]: Dùng image_tag đi em

File: app/views/orders/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <%= select_tag :status,
+          options_for_select(status_with_key),
+          class: "form-control",
+          include_blank: true %>
+        <%= submit_tag t(".search"), class: "btn btn-primary" %>
+    <% end %>
+  </div>
+  <h3><%= t ".heading" %></h3>
+
+  <div class="products-wrapper" >
```
Comment (ghost) [inline app/views/orders/index.html.erb]:
```suggestion
  <div class="products-wrapper">
```

**Review by ghost — COMMENTED** [review]
cmt

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**